lib/test.js

```diff
@@ -16,34 +16,26 @@ try {
 const functions = fs.readFileSync(path.resolve(__dirname, '../build/functions.js'), 'utf8');
 
 function getFailErrorMessage({
-  ruleName,
   functionName,
   functionsFile,
-  exampleTitle,
   exampleFile,
   expected,
   actual,
   type,
   target,
 }) {
-  return `[\u274C] FAILED: \`${ruleName}\` failed for \`${type}\` example: \`${exampleTitle}\` (target: \`${target}\`)
+  return `\u274C FAILED: ${target} => '${type}' example
   example: ${exampleFile}
   function: ${functionName} in ${functionsFile}
-  - expected + actual 
-    - ${expected}
-    + ${actual}
-  `;
-}
+  - expected + actual
 
-function getPassedErrorMessage({
-  ruleName,
-  exampleTitle,
-  type,
-  target,
-}) {
-  return `[\u2713] PASSED: \`${ruleName}\` passed for \`${type}\` example: \`${exampleTitle}\` (target: \`${target}\`)`;
+  - ${expected}
+  + ${actual || 'true'}
+  `; // If "actual" is undefined, it was supposed to be truthy (for invalid examples)
```

Comment:
anything `!== true` should be considered as failed assertion

Reply:
What I ran into was when a bad example was supposed to fail, but it passed and I only got nothing on actual, even tho the value was true

Reply:
do you have an example ?

Reply:
found it - https://github.com/apiaryio/constitutions/blob/master/lib/helpers.js#L50  -> in case assertion passed VK does not returns passed results = message is undefined